<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flight Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="flightListLabel">
          <property name="text">
           <string>Flights</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="flightList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="passengerTableLabel">
          <property name="text">
           <string>Passengers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="passengerTable">
          <column>
           <property name="text">
            <string>First Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Phone</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Seat</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveData"/>
    <addaction name="actionLoadData"/>
    <addaction name="actionExportData"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFlights">
    <property name="title">
     <string>Flights</string>
    </property>
    <addaction name="actionNewFlight"/>
    <addaction name="actionEditFlight"/>
    <addaction name="actionDeleteFlight"/>
   </widget>
   <widget class="QMenu" name="menuPassengers">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionAddPassenger"/>
    <addaction name="actionEditPassenger"/>
    <addaction name="actionRemovePassenger"/>
    <addaction name="separator"/>
    <addaction name="actionSearchPassenger"/>
   </widget>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
    <addaction name="actionFlightReport"/>
    <addaction name="actionPassengerReport"/>
    <addaction name="actionRevenueReport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFlights"/>
   <addaction name="menuPassengers"/>
   <addaction name="menuReports"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveData">
   <property name="text">
    <string>Save Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoadData">
   <property name="text">
    <string>Load Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExportData">
   <property name="text">
    <string>Export Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionNewFlight">
   <property name="text">
    <string>New Flight</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEditFlight">
   <property name="text">
    <string>Edit Flight</string>
   </property>
  </action>
  <action name="actionDeleteFlight">
   <property name="text">
    <string>Delete Flight</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAddPassenger">
   <property name="text">
    <string>Add Passenger</string>
   </property>
  </action>
  <action name="actionEditPassenger">
   <property name="text">
    <string>Edit Passenger</string>
   </property>
  </action>
  <action name="actionRemovePassenger">
   <property name="text">
    <string>Remove Passenger</string>
   </property>
  </action>
  <action name="actionSearchPassenger">
   <property name="text">
    <string>Search Passenger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFlightReport">
   <property name="text">
    <string>Flight Report</string>
   </property>
  </action>
  <action name="actionPassengerReport">
   <property name="text">
    <string>Passenger Report</string>
   </property>
  </action>
  <action name="actionRevenueReport">
   <property name="text">
    <string>Revenue Report</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui> 